jenkins/ci.suse.de/cloud-mkphyscloud-qa-test-upgrade-ui.yaml

```diff
@@ -140,7 +142,7 @@
               onadmin_upgrade_ses_to_4
             fi
 
-            export cloudsource=$cloudsource
+            export cloudsource=$original_cloudsource
```

Comment:
Why do you even need to store `original_cloudsource`? Can't you use just `cloudsource` at this point?